dmutils/apiclient/data.py

```diff
@@ -173,7 +173,12 @@ def update_user_password(self, user_id, new_password):
                         user_id, e.status_code)
             return False
 
-    def update_user(self, user_id, locked=None, active=None):
+    def update_user(self,
+                    user_id,
+                    locked=None,
+                    active=None,
+                    role=None,
+                    supplier_id=None):
```

Comment:
What's the idea with these vertical argument lists? We use them sometimes but not others. I find them quite hard to read.